README.md

```diff
@@ -39,6 +39,57 @@ lint your code before a commit, but you can always lint manually by running
 
 Please ensure new files contain our license header; this will be checked in CI as well.
 
+## GitHub App Authentication
```

Comment:
@jonkeane would you mind following these instructions to create a `conbench` org GitHub App? Then you can try the integration tests locally following the integration test instructions.

Reply:
@jonkeane Let me know when you've run the integration tests. If they pass, I could add them to CI here in this repo using your bot's creds.

Reply:
Done, and success https://github.com/conbench/benchalerts/pull/5#issuecomment-1256256190